Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,17 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  return tasks
+    .map(taskToDurations => taskToDurations.duration / 60)
+    .filter(lessThanTwoHours => lessThanTwoHours >= 2)
```

Comment:
Same goes for this parameter. An array of durations that are `lessThanTowHours` is the result you get from this filter.
I would call the item that get's passed in this filter `durationInHours`.